[2022-11-18 14:12:50,614]: Skip avahi_enable search...
[2022-11-18 14:12:50,615]: SSL is not enabled...
[2022-11-18 14:12:50,617]: [-> CONNECT           ] Connect to Host: 127.0.0.1:11083
[2022-11-18 14:12:50,620]: Some error
[2022-11-18 14:12:50,620]: Traceback message : Traceback (most recent call last):
  File "/home/jyeom/anaconda3/envs/py37tf26/lib/python3.7/site-packages/big_thing_py/manager_thing.py", line 228, in receive_staff_message_thread_func
    self._handle_staff_message(msg)
  File "/home/jyeom/big-thing-py-examples/manager_thing/lora_manager_thing/lora_manager_thing.py", line 38, in _handle_staff_message
    if 'state' in lora_device:
TypeError: argument of type 'int' is not iterable

Error: argument of type 'int' is not iterable
[2022-11-18 14:12:50,621]: Traceback message : Traceback (most recent call last):
  File "/home/jyeom/anaconda3/envs/py37tf26/lib/python3.7/site-packages/big_thing_py/manager_thing.py", line 157, in run
    raise Exception('_g_exit was set!!!')
Exception: _g_exit was set!!!

Error: _g_exit was set!!!
[2022-11-18 14:12:52,624]: Traceback message : Traceback (most recent call last):
  File "/home/jyeom/anaconda3/envs/py37tf26/lib/python3.7/site-packages/big_thing_py/manager_thing.py", line 183, in wrapup
    thread.exit()
AttributeError: 'SoPThread' object has no attribute 'exit'

Error: 'SoPThread' object has no attribute 'exit'
